My-Spec-Tutorial.package/RadioButtonGroup.extension/instance/selectedButton.st

```diff
@@ -0,0 +1,4 @@
+*My-Spec-Tutorial
+selectedButton
+
+	^ buttons detect: [ :b | b state ] ifNone: [ nil ]
```

Comment:
Did you resolve the "disappearing classes" problem? If not, please write specific reproducible steps so I can investigate further.